docs/source/topics/packaging.rst

```diff
@@ -94,54 +95,82 @@ This directory structure is then zipped up and sent to AWS Lambda during the
 deployment process.
 
 
-Psycopg2 Example
-----------------
+Cryptography Example
+--------------------
+
+Below shows an example of how to use the
+`cryptography <https://pypi.python.org/pypi/cryptography>`__ package in a
+Chalice app for the ``python3.6`` lambda environment.
+
+We're going to leverage the ``vendor/`` directory in order to use this package
```

Comment:
It might be worth making the mention that since chalice will try to build the wheel for you and you are on the right system, it may be able to work via the `requirements.txt`. I am only suggesting it because it came off to me as `cryptography` **will** require additional steps to be part of the deployment package.